### ISSUE TITLE
Can't set the default value of property to blank/none any more

### ISSUE THREAD

**Issue by coderluy** [strongloop/generator-loopback#213]
apiconnect-cli-loopback test catches the issue. Looks like a regression introduced by https://github.com/strongloop/generator-loopback/pull/200

```
apic loopback:property
$ apic loopback:property
? Select the model: Note
? Enter the property name: user
? Property type: string
? Required? No
? Default value[leave blank for none]: 
Warning: please enter the user property again. The default value provided "" is not valid for type: string
? Enter the property name:
```

**Comment by coderluy** [comment]
```$ apic --ext-version
apiconnect: v2.1.0 (API Connect: vdev)
 apiconnect-cli-apis 2.1.14 (dda3613ccaa5f612e1336c372bf120a8adf0f87d)
 apiconnect-cli-apps 2.1.16 (83c14e9a2e470888b5b6c7bcde974b591bd8e7ad)
 apiconnect-cli-auth 2.1.27 (71a0faa7661556658c2ffec2ef8e75ea5d9c3a64)
 apiconnect-cli-catalogs 2.1.5 (9e5737692eddfafee73686ec3d94c0610220cc33)
 apiconnect-cli-config 2.1.5 (e01f4d1578259eb357c804027dc10cc4d20a5ad0)
 apiconnect-cli-create 2.1.14 (74cf204378f885f8866065e56a5a0acfe462c25b)
 apiconnect-cli-drafts 2.1.13 (d851d11ea809498bc0919b8b78cce8818e6f4696)
 apiconnect-cli-edit 2.1.10 (c11b72d44a9e59ba96f98c92905d028d7748c42c)
 apiconnect-cli-loopback 2.1.17 (e88f0035f3182c0b3ee23ca1c682aae2db131803)
 apiconnect-cli-orgs 2.1.4 (43e1095fa047f50ceda63ad30ae44db660b5920c)
 apiconnect-cli-pm 2.1.14 (20d8fb2bdc5ee584ded04bb591c5cd16e65aed67)
  microgateway 1.2.17 (15332c13fe7554cdb28bf5f243d8a833af7c7ce0)
   flow-engine 1.1.0 (5b0fd2d88618b2dc772f711bf54d828f4af60b69)
 apiconnect-cli-policies 1.1.10 (b9ba41d63b20c72549b3112691f5a70bc0503180)
 apiconnect-cli-products 2.1.16 (25d8ccafa5a4cf6a5914a41ed5f3f3053ca077b1)
 apiconnect-cli-validate 2.1.12 (3f2a02dc680bb0b073eb1654ee669746daa41811)
 plugins 3.0.26

├─┬ apiconnect@2.1.0 -> /Users/ylu/apimesh/apiconnect
│ └─┬ apiconnect-cli-loopback@2.1.17
│   └── generator-loopback@1.22.24 
```

**Comment by C3realGuy** [comment]
I encountered a very similar problem with the latest version (1.23.2):
`Enter an empty property name when done.
? Property name: date
   invoke   loopback:property
? Property type: string
? Required? Yes
? Default value[leave blank for none]: 
Warning: please enter the date property again. The default value provided "" is not valid for type: string
? Property type: date
? Required? Yes
? Default value[leave blank for none]: 
Warning: please enter the date property again. The default value provided "" is not valid for type: date
` 

`slc -v
strongloop v6.0.1 (node v6.2.2)
├── strong-arc@1.8.8 (fb756f2)
├── strong-build@2.1.2 (a98f310)
├── strong-deploy@3.1.4 (9b29df0)
├── strong-mesh-models@8.1.0 (62e539b)
├── strong-pm@5.2.3
├── strong-registry@1.1.6 (f8c2339)
├── strong-start@1.3.4 (28c261d)
├─┬ strong-supervisor@3.3.3
│ └── strong-agent@2.0.4 (de9885e)
├── generator-loopback@1.23.2 (ebe6d41)
├── node-inspector@0.7.4
└── nodefly-register@0.3.3
`

**Comment by tommedema** [comment]
Same here. I guess this is not as intended, i.e. empty strings or NULL values should be possible as default values.

**Comment by bajtos** [comment]
I have sent a pull request to fix this and restore the older behaviour before #200 was landed, see https://github.com/strongloop/generator-loopback/pull/216

**Comment by bajtos** [comment]
Fixed by #216. @coderluy could you please verify?

**Comment by C3realGuy** [comment]
It's old but yeah this one's fixed (at least for me).
